src/__tests__/collections-test.js

```diff
@@ -0,0 +1,23 @@
+/* @flow strict-local */
+
+const { mapOrNull } = require('../collections');
+
+describe('mapOrNull', () => {
+  test('behaves right in happy case', () => {
+    expect(mapOrNull([10, 20], (x, i, a) => x + i + a.length)).toEqual([12, 23]);
```

Comment:
This would be an odd way to use `mapOrNull` in the wild, right, but it catches all the potential pitfalls, and quite succinctly! Staring at it, I suppose its oddness is good because it makes me go and make sure I understand `mapOrNull` well. 🙂

Reply:
:smile: